pegasus/forms/hoc_signup_2017.rb

```diff
@@ -0,0 +1,7 @@
+require pegasus_dir 'forms/hoc_signup_2014'
+
+class HocSignup2017 < HocSignup2014
+  def self.receipt
+    'hoc_signup_2017_receipt'
```

Comment:
As noted in the MD file, things should be rewritten so that this method becomes something like
```
%w(co la pe).include? @country 
  'hoc_signup_2017_receipt_es'
elsif @country == "br"
  'hoc_signup_2017_receipt_br'
else
  'hoc_signup_2017_receipt_en'
end
```